README.md

```diff
@@ -86,14 +86,14 @@ const client = new OneSignal.DefaultApi(configuration);
 ---
 ## API Reference
 
-> See the full list of [API Endpoints](DefaultApi.md).
+> See the full list of [API Endpoints](DefaultApi.MD).
 
 To make stateful changes requests should take on the following pattern:
 1. create or get an object
 2. make changes to that object
 3. pass the object to the request function to make the changes
 
-Examples of important OneSignal objects include `App`, `Notification`, `User`, and `Segment`.
+Examples of important OneSignal objects include `App`, `Notification`, `Player`, and `Segment`.
```

Comment:
Also double checking on this, it doesn't seem correct?

Reply:
same comment as above